skills/notion-import/SKILL.md

```
@@ -0,0 +1,84 @@
+description: Use when the user wants to save the previous research/answer to a local markdown file and append it to today's Notion briefing page
+argument-hint: "<topic-slug>"
+allowed-tools: Write, mcp__claude_ai_Notion__notion-search, mcp__claude_ai_Notion__notion-update-page
+---
+
+# Notion Import
+
+会話の直前の調査結果・回答内容を、ローカル md ファイルに保存しつつ、今日の Notion ブリーフィングページ末尾に追記する。
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)